test/openscap_parser/test_result_file_test.rb

```diff
@@ -416,6 +416,294 @@ def setup
     end
   end
 
+  context 'Verify the parser works the same with ARF and XCCDF files' do
+    {
+      'xccdf' => OpenscapParser::TestResultFile.new(file_fixture('ssg-rhel9-ds-xccdf.results.xml').read),
+      'arf' => OpenscapParser::TestResultFile.new(file_fixture('ssg-rhel9-ds-arf.xml').read)
+    }.each do |report_type, report|
+      context 'benchmark' do
+        test "#{report_type}_report_description" do
+          assert_match(/^This guide presents/, report.benchmark.description)
+        end
+
+        test "#{report_type}_profile_id" do
+          assert_equal 'xccdf_org.ssgproject.content_profile_anssi_bp28_enhanced', report.benchmark.profiles.first.id
+        end
+
+        context 'profiles' do
+          test "#{report_type}_profile_selected_rule_ids" do
+            assert_equal(163, report.benchmark.profiles.first.selected_rule_ids.length)
+            refute_includes(report.benchmark.profiles.first.selected_rule_ids,
+                            'xccdf_org.ssgproject.rules_group_crypto')
+            refute_includes(report.benchmark.profiles.first.selected_rule_ids,
+                            'xccdf_org.ssgproject.content_group_rule_crypto')
+            refute_includes(report.benchmark.profiles.first.selected_rule_ids,
+                            'xccdf_org.ssgproject.contentrule_group_crypto')
+            refute_includes(report.benchmark.profiles.first.selected_rule_ids,
+                            'xccdf_org.ssgproject.content_group_rule_group_crypto')
+          end
+
+          test "#{report_type}_profile_unselected_group_ids" do
+            assert_equal(190, report.benchmark.profiles.first.unselected_group_ids.count)
+            assert_includes(report.benchmark.profiles.first.unselected_group_ids,
+                            'xccdf_org.ssgproject.content_group_accounts-banners')
+            assert_includes(report.benchmark.profiles.first.unselected_group_ids,
+                            'xccdf_org.ssgproject.content_group_accounts-physical')
+            assert_includes(report.benchmark.profiles.first.unselected_group_ids,
+                            'xccdf_org.ssgproject.content_group_ftp')
+            assert_includes(report.benchmark.profiles.first.unselected_group_ids,
+                            'xccdf_org.ssgproject.content_group_gnome_media_settings')
+          end
+
+          test "#{report_type}_profile_refined_values" do
+            # rubocop:disable Layout/LineLength
```

Comment:
It should be possible to edit these lines to get passed the LineLength errors without disabling the check

Reply:
I've solved the formatting problems. Should I enable the line length check on line 164 as well?

Reply:
Yeah, sure since it should be a quick change. This looks good now. Thank you!